flo-freezer/src/test/java/com/spotify/flo/freezer/PersistingContextTest.java

```diff
@@ -101,6 +105,46 @@ public void deserializeShouldPropagateSerializationExceptions() {
     PersistingContext.deserialize(new ByteArrayInputStream("foobar".getBytes()));
   }
 
+  @Test
+  public void shouldProvideDetailedDebugInformationOnSerializationExceptions() throws ReflectiveOperationException {
```

Comment:
TODO: Move serialization code and tests from `flo-freezer` module to `flo-workflow` to avoid duplication of code. Separate PR.

Reply:
https://github.com/spotify/flo/pull/182